docs/GETTING_STARTED_BEGINNER.md

```
@@ -0,0 +1,1109 @@
+# Omega_KG Beginner's Guide
+
+**Getting Started with Knowledge Graph Management**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)